test/CadlRanchProjects/client/structure/default/tspconfig.yaml

```diff
@@ -1,3 +1,4 @@
 options:
   "@azure-tools/typespec-csharp":
-    namespace: Client.Structure.Service.Default
\ No newline at end of file
+    namespace: Client.Structure.Service.Default
+    generate-sample-project: false
```

Comment:
because of [this issue](https://github.com/Azure/autorest.csharp/issues/4151), I have to disable sample generation for this project.
[This PR](https://github.com/Azure/autorest.csharp/pull/4082) might fix the issue for this test case and we could reenable when it merges.